### PR TITLE
feat(webpack): settings webpack and create index.html

### PR DESCRIPTION
**Pull request by Fryly**

### CODE REVIEW

**Review by evxeen — DISMISSED** [review]

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/******/ (() => { // webpackBootstrap
+/******/ 	var __webpack_modules__ = ({
+
+/***/ "./src/index.js":
+/*!**********************!*\
+  !*** ./src/index.js ***!
+  \**********************/
+/***/ (() => {
+
+eval("console.log(\"poshel naxyi\");\n\n//# sourceURL=webpack://webpack-demo/./src/index.js?");
```
Comment (NikDoe) [inline dist/main.js:18]: judgmental behavior

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "webpack-demo",
```
Comment (NikDoe) [inline package.json]: у тебя проект не webpack-demo

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import './styles/main.scss';
+
+console.log("poshel naxyi");
```
Comment (NikDoe) [inline src/index.js]: judgmental behavior

**Review by NikDoe — COMMENTED** [review]
не прошёл ревью

**Review by NikDoe — APPROVED** [review]
good job Maxim

**Review by evxeen — APPROVED** [review]

**Pull request merged**